tests/generators/bls/main.py

```diff
@@ -355,7 +358,138 @@ def case05_aggregate_verify():
     }
 
 
-def create_provider(handler_name: str,
+def case06_eth2_aggregate_pubkeys():
+    aggregate_pubkey = spec.eth2_aggregate_pubkeys(PUBKEYS)
+    assert aggregate_pubkey == milagro_bls._AggregatePKs(PUBKEYS)
+    yield f'eth2_aggregate_pubkeys_some_pubkeys', {
+        'input': [encode_hex(pubkey) for pubkey in PUBKEYS],
+        'output': encode_hex(aggregate_pubkey),
+    }
+
+    # Invalid pubkeys -- len(pubkeys) == 0
+    expect_exception(spec.eth2_aggregate_pubkeys, [])
+    expect_exception(milagro_bls._AggregatePKs, [])
+    yield f'eth2_aggregate_pubkeys_empty_list', {
+        'input': [],
+        'output': None,
+    }
+
+    # Invalid pubkeys -- [ZERO_PUBKEY]
+    expect_exception(spec.eth2_aggregate_pubkeys, [ZERO_PUBKEY])
+    expect_exception(milagro_bls._AggregatePKs, [ZERO_PUBKEY])
+    yield f'eth2_aggregate_pubkeys_na_pubkey', {
+        'input': [encode_hex(ZERO_PUBKEY)],
+        'output': None,
+    }
+
+    # Invalid pubkeys -- G1 point at infinity
+    expect_exception(spec.eth2_aggregate_pubkeys, [Z1_PUBKEY])
+    expect_exception(milagro_bls._AggregatePKs, [Z1_PUBKEY])
+    yield f'eth2_aggregate_pubkeys_infinity_pubkey', {
+        'input': [encode_hex(Z1_PUBKEY)],
+        'output': None,
+    }
```

Comment:
add an invalid test case for G2 point?

Reply:
just thinking more broadly about possible easy to make errors we could catch here...

Reply:
I'm afraid it would break the test format doc:
> - `BLS Pubkey` here is encoded as a string: hexadecimal encoding of 48 bytes (96 nibbles), prefixed with `0x`.

I tried it on my end and it works correctly (return INVALID). The wrong length input should be caught at deserialization.